cli/tests/data/package_build/cassandra/marathon.json.mustache

```diff
@@ -0,0 +1,174 @@
+
```

Comment:
I just plopped cassandra in there. We need a package that can run correctly. If you have any suggestion for what other package we could use, I'd be happy to use that.